wopiserver/templates/deployment.yaml

```diff
@@ -32,6 +32,8 @@ spec:
             - name: {{ include "wopiserver.fullname" . }}-confdir
               mountPath: /etc/wopi/iopsecret
               subPath: iopsecret
+            - name: {{ include "wopiserver.fullname" . }}-secret
+              mountPath: /var/run/secrets
```

Comment:
Use the `{{ include "wopiserver.fullname" . }}-confdir` + `mountPath: /var/run/secrets/codimd_apikey` + `subPath: codimd_apikey` comination to avoid double mounts. In the future, if we have an arbitrary number of secrets to be split from the config and mounted in the same `/var/run/secrets` directory we might split the projected volume below.

Reply:
Thanks, I must say that I did a bit of guesswork :-P now it should be OK but I still did not run any test/linting